.pylintrc

```diff
@@ -1,519 +0,0 @@
-[MAIN]
```

Comment:
Why have you removed the `.pylintrc` file? Is it a mistake or a thoughtful action?

Reply:
!!! To restore !!!